SYCL/InvokeSimd/Spec/simd_size/simd_mismatch_error.cpp

```diff
@@ -0,0 +1,52 @@
+// TODO: enable on Windows once driver is ready
+// REQUIRES: gpu && linux
+// UNSUPPORTED: cuda || hip
+//
+// TODO: enable when Jira ticket resolved
+// XFAIL: gpu
+//
+// Check that full compilation works:
+// RUN: %clangxx -fsycl -fno-sycl-device-code-split-esimd -Xclang -fsycl-allow-func-ptr %s -o %t.out
+// RUN: env IGC_VCSaveStackCallLinkage=1 IGC_VCDirectCallsOnly=1 %GPU_RUN_PLACEHOLDER %t.out
+
+/* Tests invoke_simd support in the compiler/headers
+ * Test checks an error message when subgroup size mismatches invoke_simd
+ * return type (simd<T, M> or simd_mask<T, M>) size
```

Comment:
Minor comment: it verifies not only 'return', but also 'arguments' of invoke_simd target function.
```suggestion
 * The test checks that compiler emits a meaningful and user friendly error message
 * when the target function of invoke_simd has arguments or returns of simd<T,N> or
 * simd_mask<T,N> type and N is not equal to subgroup size.
```